lms/static/scripts/frontend_apps/components/ContentSelector.js

```diff
@@ -151,7 +151,6 @@ export default function ContentSelector({
     case 'vitalSourceBook':
       dialog = (
         <BookPicker
-          authToken={authToken}
```

Comment:
This makes me happy. I'm not the most familiar-est with all of the LMS frontend code, but every time I see the `authToken` being passed between components I feel a little fizzy and weird. It's nice to see it getting pulled into services as we go here...